server/src/testFixtures/java/io/crate/testing/JoinPairAssert.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Licensed to Crate.io GmbH ("Crate") under one or more contributor
+ * license agreements.  See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.  Crate licenses
+ * this file to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial agreement.
+ */
+
+package io.crate.testing;
+
+
+import static io.crate.testing.Asserts.assertThat;
+
+import org.assertj.core.api.AbstractAssert;
+
+import io.crate.analyze.relations.JoinPair;
+import io.crate.metadata.RelationName;
+
+public class JoinPairAssert extends AbstractAssert<JoinPairAssert, JoinPair> {
+
+    protected JoinPairAssert(JoinPair actual) {
+        super(actual, JoinPairAssert.class);
+    }
+
+    public JoinPairAssert isEqualTo(JoinPair expected) {
+        isNotNull();
+        assertThat(actual.joinType()).isEqualTo(expected.joinType());
+        assertThat(actual.left()).isEqualTo(expected.left());
+        assertThat(actual.right()).isEqualTo(expected.right());
```

Comment:
Is this used somewhere? sorry if I missed a usage, but couldn't find any(

Reply:
No it's not used, it's only for completeness to make the assert behave normally.

Reply:
Wouldn't it make sense to implement. equals on the JoinPair instead? 

I think in general we should reserve custom asserts to cases where the assertion contains some more logic. 
If it only saves a tiny amount of typing there's a good chance we'll have mixed variants all over the place

Reply:
`equals` is already implemented on JoinPairs, so this method is useless. I wasn't aware that equality works out-of-the-box like this with assertj. I will remove it.